src/Octoshift/AdoClient.cs

```diff
@@ -133,6 +148,44 @@ public virtual async Task<JArray> GetWithPagingAsync(string url, string continua
             return data;
         }
 
+        public virtual async Task<IEnumerable<T>> GetWithPagingTopSkipAsync<T>(string url, Func<JToken, T> selector) => await GetWithPagingTopSkipAsync(url, 0, selector);
```

Comment:
Is this basically same as `GetWithPagingAsync` only with an additional `selector`? If so it'd be better to make this an overload of it.

Reply:
it is not. I've learned that some ADO API's do paging differently than others. The previous GetWithPagingAsync uses a continuation token returned as a HTTP header to get the next page. This uses the top/skip query params to perform paging. Depending on which ADO endpoint you are calling you will need to use one or the other paging method.